l10n_th_multicurrency_revaluation_operating_unit/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_wizard_currency_revaluation_user,access_wizard_currency_revaluation_user,l10n_th_multicurrency_revaluation.model_wizard_currency_revaluation,base.group_user,1,1,1,1
```

Comment:
why we add this permission? it will duplicate with l10n_th_multicurrency_revaluation?